src/main/kotlin/ch/hsr/dsl/dwrtc/websocket/Main.kt

```diff
@@ -3,22 +3,30 @@ package ch.hsr.dsl.dwrtc.websocket
 import ch.hsr.dsl.dwrtc.signaling.ClientService
 import ch.hsr.dsl.dwrtc.util.*
 import io.javalin.Javalin
+import io.javalin.staticfiles.Location
+import mu.KotlinLogging
 import org.eclipse.jetty.server.Server
 import org.eclipse.jetty.server.ServerConnector
 
 /** Main start point. Fires up the Webserver. */
 fun main(args: Array<String>) {
+    val logger = KotlinLogging.logger {}
     val app = Javalin.create()
-            // .enableStaticFiles("src/main/resources/public/", Location.EXTERNAL)
-            .enableStaticFiles("/public").server {
-                Server().apply {
-                    connectors = arrayOf(ServerConnector(this).apply {
-                        this.host = config[WEBSERVER_IP]
-                        this.port = config[WEBSERVER_PORT]
-                    })
-                }
-            }
-            .start()
+    if (config.getOrElse(PACKED_RESOURCES, true)) {
+        logger.info { "using packed web resources" }
+        app.enableStaticFiles("/public")
```

Comment:
We will need to change those to `/web/public` as soon as the JS refactoring is merged.